src/components/Notification/Notification.tsx

```diff
@@ -30,6 +30,8 @@ type BasePropsWithoutAlignText = Omit<BaseProps, "alignText">;
 type BasePropsWithoutCentered = Omit<BaseProps, "centered">;
 
 export interface NotificationProps extends BasePropsWithoutAlignText {
+  /** Label used to describe the `Notification`'s aside HTML element. */
+  ariaLabel?: string;
```

Comment:
We've gone out of our way to remove explicit `aria-label` props from other components.  I'm wondering if there is another way we can handle this here.  The pattern we've used elsewhere is to require some type of title text, which would in turn be used to populate `aria-label`, and then control the visibility of the title element with another boolean prop.  I feel if we could that route here, then a better prop name would be needed for `  notificationHeading`.  What do you think?

Reply:
Yeah, that's a good point for our DS pattern. I was thinking that it might make sense to use the text from the `notificationHeading` prop but it's not always applicable and the heading is not required. I was trying to follow the description from [DSD-209](https://jira.nypl.org/browse/DSD-209) but also what's in [Metronome](http://themetronome.co/components/notification/). Specifically, the example aria-labels in Metronome have "Library-wide notification" and "Location notification" which are good but perhaps not the best to display in the actual `notificationHeading` prop. 

So I was thinking of the case where it might be something like
```
<Notification
  ariaLabel="Library-wide notification"
  notificationHeading="Snow day closure"
  notificationContent="All NYPL branches will have a delay opening on ...."
/>
```
and the case where there are multiple `Notifications` with the same heading string but different content. The aria-labels for each `aside` _should_ be unique.

We can use the `notificationHeading` prop and add more patterns for users to implement which works for me.

Reply:
I don't. Let's set up a call about it soon.